src/webdemo-api.js

```diff
@@ -173,6 +206,26 @@ function executeCode(url, code, compilerVersion, targetPlatform, args, hiddenDep
   }).then(response => response.json())
 }
 
+function executeCodeStreaming(url, code, compilerVersion, targetPlatform, args, hiddenDependencies, callback) {
+  const body = createBodyForCodeExecution(code, compilerVersion, targetPlatform, args, hiddenDependencies)
+
+  jsonpipe.flow(url + targetPlatform.id, {
+    success: data => {
+      callback({'data': data})
+    },
+    complete: () => {
+      callback({'done': true})
+    },
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/x-www-form-urlencoded;charset=UTF-8',
```

Comment:
Not necessary to use the `application/x-www-form-urlencoded` in the new API because there is another endpoint on the server-side

Reply:
Using `application/json` for streaming would require copy-pasting `createBodyForCodeExecution` function. I believe that it would be much easier to switch to `appliction/json` when old server is no longer used.